src/test/rpc/NoRipple_test.cpp

```diff
@@ -213,9 +214,15 @@ class NoRipple_test : public beast::unit_test::suite
     void run ()
     {
         testSetAndClear();
-        testNegativeBalance();
-        testPairwise();
-        testDefaultRipple();
+
+        auto withFeatsTests = [this](auto&&... fs) {
+            testNegativeBalance();
```

Comment:
I believe these three calls are missing the `(fs...)`, so they perform exactly the same tests three times.